src/executables/utility.h

```diff
@@ -704,6 +705,21 @@ typedef enum
   TDE_MSG_USAGE = 60
 } MSGCAT_TDE_MSG;
 
+/* Message id in the set MSGCAT_UTIL_SET_TDE */
+typedef enum
+{
+  FLASHBACK_MSG_DBA_PASSWORD = 1,
```

Comment:
It has to be used instead of `TDE_MSG_DBA_PASSWORD`.

Reply:
Oh, you are right. I changed it.